assets/component-price.css

```diff
@@ -64,10 +64,15 @@
 }
 
 .price--sold-out .price__badge-sold-out,
-.price--on-sale .price__badge-sale {
+.price--on-sale .price__badge-sale,
+.volume-pricing--sale-badge .price__badge-sale {
   display: inline-block;
 }
 
+.volume-pricing--sale-badge .price__badge-sale {
```

Comment:
When youre not logged in, the price doesnt show when its unavailable (i havent tested sold out yet)

https://screenshot.click/19-53-7v1pu-ak2e9.png

Reply:
I think we never showed the price when the variant is unavailable, this is expected as far as I can remember.

Reply:
I always thought the same but it was done like this to avoid layout shifting.

Reply:
I totally agree @sofiamatulis... this might be a part of a larger issue though, #2870. I feel like this should qualify as a fast-follow though. 

It is also presently something we're doing on current versions of Dawn, like @eugenekasimov mentioned... I believe to account for layout shift?

https://screenshot.click/Current-Dawn-Price-block-hidden.png

Reply:
I think we can leave as is, this was not introduced in this PR